src/shaders/standard.js

```diff
@@ -10,29 +10,58 @@ var texturePromises = {};
  */
 module.exports.Component = registerShader('standard', {
   schema: {
+    aoMap: {default: ''},
```

Comment:
Aframe aimes at being more amicable for non graphics people. Let's use full descriptive names